plugins/gradle/build.gradle

```diff
@@ -44,6 +45,28 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.11'
 }
 
+gradlePlugin {
+    website = "https://mobivm.github.io"
+    vcsUrl = "https://github.com/MobiVM/robovm"
+    plugins {
+        // Users can apply this in plugins block with the marker artifact id.
+        create("robovmNew") {
+            id = "com.mobidevelop.robovm"
+            implementationClass = "org.robovm.gradle.RoboVMPlugin"
+            displayName = "RoboVM Gradle Plugin"
+            description = "The RoboVM Gradle Plugin provides a way to build RoboVM apps using Gradle."
+        }
+
+        // Legacy plugin id, to compat users applying this plugin via buildscript classpath.
+        create("robovmLegacy") {
+            id = "robovm"
```

Comment:
I've published a snapshot to `MavenLocal`, and verified this could be applied like the way before. After publishing a new release, we can note the plugin marker way in changelog and readme.

Reply:
But seems something wrong with the namespace:

```
tree .m2/repository
.m2/repository
├── com
│   └── mobidevelop
│       └── robovm
│           ├── com.mobidevelop.robovm.gradle.plugin
│           │   ├── 2.3.21-SNAPSHOT
│           │   │   ├── com.mobidevelop.robovm.gradle.plugin-2.3.21-SNAPSHOT.pom
│           │   │   ├── com.mobidevelop.robovm.gradle.plugin-2.3.21-SNAPSHOT.pom.asc
│           │   │   └── maven-metadata-local.xml
│           │   └── maven-metadata-local.xml
│           └── robovm-gradle-plugin
│               ├── 2.3.21-SNAPSHOT
│               │   ├── maven-metadata-local.xml
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT-javadoc.jar
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT-javadoc.jar.asc
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT-sources.jar
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT-sources.jar.asc
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT.jar
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT.jar.asc
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT.module
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT.module.asc
│               │   ├── robovm-gradle-plugin-2.3.21-SNAPSHOT.pom
│               │   └── robovm-gradle-plugin-2.3.21-SNAPSHOT.pom.asc
│               └── maven-metadata-local.xml
└── robovm
    └── robovm.gradle.plugin
        ├── 2.3.21-SNAPSHOT
        │   ├── maven-metadata-local.xml
        │   ├── robovm.gradle.plugin-2.3.21-SNAPSHOT.pom
        │   └── robovm.gradle.plugin-2.3.21-SNAPSHOT.pom.asc
        └── maven-metadata-local.xml

11 directories, 20 files
```

Reply:
if you roll back robovm.property and remove legacy section -- it should work